lms/security.py

```diff
@@ -159,11 +160,9 @@ def identity(self, request):
 
         permissions = [Permissions.LTI_LAUNCH_ASSIGNMENT, Permissions.API]
 
-        if any(
```

Comment:
This method of checking roles is outdated.

Reply:
:+1: